src/main/java/org/opensearch/sdk/BaseExtensionRestHandler.java

```diff
@@ -127,4 +134,58 @@ protected ExtensionRestResponse createJsonResponse(RestRequest request, RestStat
             return new ExtensionRestResponse(request, status, fieldName + ": " + responseStr);
         }
     }
+
+    /**
+     * Returns a String message of the detail of any unrecognized error occurred
+     *
+     * @param request The request that caused the exception
+     * @param invalids invalid strings
+     * @param candidates candidates involved
+     * @param detail
```

Comment:
This needs a description of what it actually is.  Looking at line 154 it appears to be "a noun describing the category or type of invalid strings to be passed".    Also apparently it is supposed to be singular.   TLDR, I couldn't tell from reading this javadoc what I am supposed to do with this method.

Reply:
Updated